pkg/controller/release/scheduler_test.go

```diff
@@ -327,17 +498,19 @@ func TestCreateAssociatedObjectsDuplicateInstallationTargetSameOwner(t *testing.
 	expected.Status.Conditions = []shipper.ReleaseCondition{
 		{Type: shipper.ReleaseConditionTypeScheduled, Status: corev1.ConditionTrue},
 	}
-	expectedActions := buildExpectedActions(release.GetNamespace(), expected)
+	expectedActions := buildExpectedActions(expected.DeepCopy(), []*shipper.Cluster{cluster})
+	// an installationtarget exists and contains the spec we expect, no action required
+	expectedActions = filterActions(expectedActions, []string{"update", "create"}, []string{"traffictargets", "capacitytargets"})
 
 	c, clientset := newScheduler(release, fixtures)
-	if err := c.scheduleRelease(); err != nil {
+	if _, err := c.ScheduleRelease(release.DeepCopy()); err != nil {
 		t.Fatal(err)
 	}
 
 	filteredActions := filterActions(
 		clientset.Actions(),
 		[]string{"update", "create"},
-		[]string{"releases", "installationtargets", "traffictargets", "capacitytargets"},
+		[]string{"installationtargets", "traffictargets", "capacitytargets"},
```

Comment:
Don't we have any updates on the Release object we should check after all target objects have been created?

Reply:
Indeed, this was missing from my original commits, fixed in the newer version.